app/src/main/java/com/google/jetpackcamera/ui/JcaApp.kt

```diff
@@ -26,35 +25,39 @@ import com.google.accompanist.permissions.ExperimentalPermissionsApi
 import com.google.accompanist.permissions.isGranted
 import com.google.accompanist.permissions.rememberPermissionState
 import com.google.jetpackcamera.feature.preview.PreviewScreen
+import com.google.jetpackcamera.feature.preview.PreviewViewModel
 import com.google.jetpackcamera.settings.SettingsScreen
 import com.google.jetpackcamera.ui.Routes.PreviewRoute
 import com.google.jetpackcamera.ui.Routes.SettingsRoute
 
 @OptIn(ExperimentalPermissionsApi::class)
 @Composable
-fun JcaApp() {
+fun JcaApp(onPreviewViewModel: (PreviewViewModel) -> Unit) {
```

Comment:
Let's add a TODO to remove this callback once still capture is enabled. These tests should rely on end-to-end flows when possible.